pkg/rpmdb_test.go

```diff
@@ -95,23 +108,25 @@ func TestPackageList(t *testing.T) {
 			got, err := db.ListPackages()
 			require.NoError(t, err)
 
-			// They are tested in another function.
+			pkgList := make([]*commonPackageInfo, 0, len(got))
 			for _, g := range got {
-				g.BaseNames = nil
-				g.DirIndexes = nil
-				g.DirNames = nil
-				g.FileSizes = nil
-				g.FileDigests = nil
-				g.FileModes = nil
-				g.FileFlags = nil
-				g.UserNames = nil
-				g.GroupNames = nil
-				g.Provides = nil
-				g.Requires = nil
+				pkgList = append(pkgList, &commonPackageInfo{
```

Comment:
Do we need to use `commonPackageInfo`? I thought we could use `PackageInfo` with necessary fields only as I suggested [here](https://github.com/knqyf263/go-rpmdb/pull/38/files#r1178838003).

Reply:
I'm now limiting it's use to within rpmdb_testcase_test.go. I hope this is more in line with what you are expecting.

Reply:
You pushed forcibly, and the previous changes have gone now. I thought your approach was good, but I just 
I suggested using `PackageInfo` instead of `commonPackageInfo` to compare only the fields you defined in `commonPackageInfo`. Or am I missing something?